src/stripeJavaLanguageClient/syntaxLanguageClient.ts

```diff
@@ -31,46 +29,31 @@ export class SyntaxLanguageClient {
       },
     });
 
-    const lsPort = process.env[SYNTAXLS_CLIENT_PORT];
-    if (!serverOptions && lsPort) {
```

Comment:
If we get rid of this, does that mean we could be starting up multiple java language servers?

Reply:
actually yes if its in hybrid mode, then both standard and syntax servers could be started